app/views/school/devices/can_order.html.erb

```diff
@@ -12,9 +12,12 @@
 
     <%= render DeviceCountComponent.new(school: @school) %>
 
-    <p class="govuk-body">Only order devices for children who cannot attend school and do not have access to a laptop or tablet.</p>
-    <p class="govuk-body">We’re working hard to get devices delivered within 5 working days.</p>
-    <p class="govuk-body">Ordering will close when your school reopens.</p>
+    <p class='govuk-body'>Order your devices now. Do not delay.</p>
+    <p class='govuk-body'>Most devices will be delivered within 2 working days.</p>
+    <% if @school.std_device_allocation.available_devices_count > 149 %>
```

Comment:
Do we need a test for this?

We might also want to use the safe navigation operator in case the school doesn't have an allocation:
```suggestion
    <% if @school&.std_device_allocation&.available_devices_count > 149 %>
```